### PR TITLE
vibration

### LINKED ISSUE #2: Add vibrate when finish zeker so user know that he finished it

### PR DESCRIPTION
**Pull request by hamza94max**

### CODE REVIEW

File: app/src/main/java/com/mahmoudmabrok/azakri/feature/display/DisplayAzkar.java [hunk 1]
```diff
@@ -93,6 +98,15 @@ public void onTick(long millisUntilFinished) {
 
             @Override
             public void onFinish() {
+                Vibrator v = (Vibrator) getSystemService(Context.VIBRATOR_SERVICE);
```
Comment (MahmoudMabrok) [inline app/src/main/java/com/mahmoudmabrok/azakri/feature/display/DisplayAzkar.java]: we want to add vibrate after each finish to zeker not all zeker

Comment (MahmoudMabrok) [reply app/src/main/java/com/mahmoudmabrok/azakri/feature/display/DisplayAzkar.java]: also try to add it as HELPER CLASS so it can be used in other places

**Review by MahmoudMabrok — CHANGES_REQUESTED** [review]
see comments

File: app/src/main/java/com/mahmoudmabrok/azakri/feature/display/DisplayAzkar.java (hunk 1 above)
Comment (hamza94max) [reply app/src/main/java/com/mahmoudmabrok/azakri/feature/display/DisplayAzkar.java]: Done @MahmoudMabrok

**Review by MahmoudMabrok — APPROVED** [review]

**Pull request merged**